continuous_integration/appveyor/run_test.bat

```diff
@@ -26,4 +26,8 @@ echo %time%
 echo Success!
 echo %time%
 echo.
-echo ---------------------------------
+
+echo ------- Running py.test ---------
+echo.
+cd %APPVEYOR_BUILD_FOLDER%
+python runtests.py
```

Comment:
Please move this to `appveyor.yml` to have the same configuration as in Travis:

https://github.com/spyder-ide/spyder/blob/master/.travis.yml#L52